migrations/versions/e46b30dd3ff5_automatic_collections.py

```diff
@@ -0,0 +1,65 @@
+"""automatic_collections
+
+Revision ID: e46b30dd3ff5
+Revises: 9e33fa16ba82
+Create Date: 2023-02-06 11:44:29.000880
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = 'e46b30dd3ff5'
+down_revision = '9e33fa16ba82'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # TODO: Maybe add some CHECK constraints around some of these fields? Or just
+    # enforce in code upon creation.
+    op.create_table(
+        "automatic_collection_definition",
+        sa.Column(
+            'collection_id',
+            sa.Integer,
+            sa.ForeignKey('collections.id', ondelete='CASCADE'),
+            primary_key=True,
+        ),
+        sa.Column(
+            'query',
+            sa.ARRAY(sa.String),
+        ),
+        sa.Column(
+            'sort',
+            sa.ARRAY(sa.String),
+        ),
+        sa.Column(
+            'filter',
+            sa.ARRAY(sa.String),
+        ),
+        sa.Column(
+            'limit',
+            sa.Integer,
+            nullable=False,
+        ),
```

Comment:
I think that at the very least these columns should have CHECK constraints to specify the filters that are allowed and a defined limit to the number of works/editions that can be in an automated collection.

Reply:
Yea, I actually think I'm going to move these array columns to JSONB anyway, and I can implement check constraints on that.

For the `limit` clause, I'm not as concerned about limiting that, but lemme think about that. Or maybe I could just err on the side of a really high default, but I think implementing pagination properly should avoid us overwhelming ES.